composer.json

```diff
@@ -91,7 +91,7 @@
         "neos/redirecthandler-neosadapter": "Adds automatic redirects for moved/renamed pages"
     },
     "scripts": {
-        "lint:phpcs": "../../bin/phpcs --config-set ignore_warnings_on_exit 1; ../../bin/phpcs --colors --standard=PSR12 ./Neos.ContentGraph.DoctrineDbalAdapter/src ./Neos.ContentGraph.PostgreSQLAdapter/src ./Neos.ContentRepository.BehavioralTests/Classes ./Neos.ContentRepository.Core/Classes ./Neos.EventSourcedContentRepository.LegacyApi/Classes ./Neos.EventSourcedNeosAdjustments/Classes ./Neos.Neos/Classes",
+        "lint:phpcs": "../../bin/phpcs --colors --standard=PSR12 --exclude=Generic.Files.LineLength ./Neos.ContentGraph.DoctrineDbalAdapter/src ./Neos.ContentGraph.PostgreSQLAdapter/src ./Neos.ContentRepository.BehavioralTests/Classes ./Neos.ContentRepository.Core/Classes ./Neos.EventSourcedContentRepository.LegacyApi/Classes ./Neos.EventSourcedNeosAdjustments/Classes ./Neos.Neos/Classes",
```

Comment:
By excluding the line length check, no warnings were left over so we can skip the `ignore_warnings_on_exit` flag IMO to strive for a clean output

Reply:
I'd like to keep the warning, because it's a plausible way to implement PSR-12 soft limits. They can be ignored, but if anyone feels obliged to clean up, they should be encouraged to do so.

Reply:
mh, but it makes the output so verbose and I wouldn't have seen the `$_*`-warning above.
What do you think of a separate script (e.g. `lint:phpcs-psr12`) that contains the line length warnings?

Reply:
I adjusted the scripts as suggested to give this a shot. WDYT?